app/models/event.rb

```diff
@@ -10,6 +10,7 @@ class Event < ActiveRecord::Base
     authenticator_enabled: 6,
     authenticator_disabled: 7,
     account_verified: 8,
+    usps_mail_sent: 9,
```

Comment:
Cool. I did this and added a spanish translation as well. Normally I have been putting `NOT YET TRANSLATED`, but I knew this one so I went for it. Can put default text in there if that is outside of the process